tests/test_posting_models.py

```diff
@@ -104,6 +104,96 @@ def test_fail_tracking_code_invalid_range_generator():
         posting.TrackingCode.create_range("DL74668654 BR", "DL74668650 BR")  # end < start
 
 
+def test_receipt_basic():
+    receipt = posting.Receipt(number=1234, post_date='20160812', value='18.00')
+    assert receipt.number == 1234
+    assert receipt.real_post_date == '20160812'
+    assert receipt.real_value == '18.00'
+    assert isinstance(receipt.post_date, date)
+    assert receipt.post_date.day == 12
+    assert receipt.post_date.month == 8
+    assert receipt.post_date.year == 2016
+    assert receipt == posting.Receipt(
+        number=1234,
+        post_date='20160812',
+        value='18.00'
+    )
+    assert receipt != posting.Receipt(
+        number=1235,
+        post_date='20160812',
+        value='18.00'
+    )
+    assert receipt != posting.Receipt(
+        number=1234,
+        post_date='20160813',
+        value='18.00'
+    )
+    assert receipt != posting.Receipt(
+        number=1235,
+        post_date='20160813',
+        value='19.00'
+    )
+
+    fake_date = date(year=2016, month=8, day=12)
+    repr_ = (
+        '<Receipt('
+        'number={number}, '
+        'post_date={post_date}, '
+        'value={value}'
+        ')>'.format(
+            number=1234,
+            post_date=fake_date,
+            value=Decimal('18.00')
+        )
+    )
+    assert repr(receipt) == repr_
+    assert isinstance(
+        posting.Receipt(
+            number=1234,
+            post_date=fake_date,
+            value='18.00'
+        ).post_date, date
+    )
+    assert isinstance(
+        posting.Receipt(
+            number=1234,
+            post_date=fake_date,
+            value='18.00'
+        ).value, Decimal
+    )
```

Comment:
dúvida: pq não esta em testes separados o teste de representation, comparação do equals...?

Reply:
Acho que foi no auto, @erikhenrique. Você acha melhor separar? :thinking:

Reply:
Gente, temos um serviço que não esta funcionando pq esta dependendo dessa lib.
Vamos agilizar, se não for demorar pra fazer essa alteração, faz. Se for demorar muito podemos postergar isso.

Reply:
Tem que colocar em Prod em poucos dias, se a Renata desenrolar o número de contrato, e ainda não conseguimos fechar o teste integrado 100%

Reply:
acho que sim, se quebrar saberemos exatamente o que esta quebrando, até mesmo pelo nome que ficara no metodo do teste.
Aqui se este teste quebrar tem que ver em qual passo.

Reply:
o `test_post_info_basic` é a mesma coisa :)